src/hooks/useDownShift.ts

```diff
@@ -24,7 +24,11 @@ const useDownShift: UseDownShift = ({ setQuery, items, onSubmit, cioClient, prev
     itemToString: (item) => item?.value || '',
     onSelectedItemChange({ selectedItem }) {
       if (selectedItem) {
-        setQuery(selectedItem.value || '');
+        if (selectedItem?.section === 'Search Suggestions') {
+          setQuery(selectedItem.value);
+        } else {
+          setQuery('');
+        }
```

Comment:
**Suggestion**: I think it's better to keep the search input as it is, without cleaning it.

If the customer wants to add a special behavior, it's always possible to add an on-click listener. But it would be impossible to remove this "clear input" behavior.

```suggestion
        if (selectedItem?.section === 'Search Suggestions') {
          setQuery(selectedItem.value);
        }
```

Reply:
I like that